server/pbench/bin/index-pbench

```diff
@@ -852,11 +890,20 @@ class ToolData(object):
                 try:
                     # unix seconds since epoch timestamp
                     ts = datetime.utcfromtimestamp(item['@timestamp'])
+                except TypeError:
+                    # assume that item[@timestamp] is already in the expected
+                    # format.
+                    self.counters['json_doc_timestamp_not_validated'] += 1
+                except KeyError:
+                    # missing timestamps
+                    #
+                    # FIXME: We should consider logging the first record with
+                    # missing timestamps, and then count the rest and report
+                    # the count with the summary of how the indexing went.
+                    self.counters['json_doc_missing_timestamp'] += 1
```

Comment:
So all these counters are accumulated but not reported anywhere? Ah, no I see: they are stored in opctx and dumped on stdout.